.github/workflows/c3pm.yml

```diff
@@ -0,0 +1,37 @@
+name: c3pm CI
+
+on:
```

Comment:
are `on` tags useful with current values ?

Reply:
I think we can remove it indeed